tests/testthat/test-use_lintr.R

```diff
@@ -4,24 +4,36 @@ test_that("use_lintr works as expected", {
   lintr_file <- use_lintr(path = tmp)
   expect_true(file.exists(lintr_file))
 
+  # check that newly created file is in the root directory
+  expected_lintr_file_path <- file.path(
+    normalizePath(tmp), ".lintr",
+    fsep = if (.Platform$OS.type == "unix") "/" else "\\"
+  )
+  expect_identical(normalizePath(lintr_file), expected_lintr_file_path)
```

Comment:
You can use the winslash arg of normalizePath here

Reply:
That's what I had initially, but it didn't work.

Note that `fsep` is passed to `file.path()`, and not `normalizePath()`.

Reply:
You didn't use `normalizePath(..., winslash = "/")` everywhere AFAICT. That's what I'm suggesting.

Reply:
That worked! Thanks.

Looking back, it didn't work for me before because, as you noted, I wasn't using it everywhere.